client/jetpack-cloud/sections/agency-dashboard/downtime-monitoring/configure-email-notification/add-new-email-modal.tsx

```diff
@@ -165,16 +167,20 @@ export default function AddNewEmailModal( {
 					</FormFieldset>
 				) }
 				<div className="notification-settings__footer">
-					{ validationError && (
-						<div className="notification-settings__footer-validation-error">
-							{ validationError }
-						</div>
-					) }
 					<div className="notification-settings__footer-buttons">
 						<Button onClick={ toggleModal } aria-label={ translate( 'Cancel' ) }>
 							{ showCodeVerification ? translate( 'Later' ) : translate( 'Cancel' ) }
 						</Button>
-						<Button disabled={ false } type="submit" primary aria-label={ translate( 'Verify' ) }>
+						<Button
+							disabled={
+								! emailItem.name ||
+								! emailItem.email ||
+								( showCodeVerification && ! emailItem.code )
+							}
+							type="submit"
+							primary
+							aria-label={ translate( 'Verify' ) }
```

Comment:
I think we  don't need `aria-label` here since the button already has a text content.